### PR TITLE
Eval, full arguments support, fixes for getters

### PR DESCRIPTION
**Pull request by baxtersa**

### CODE REVIEW

**Comment by rachitnigam** [comment]
This branch has fixes to several things including the CLI and webpacking. If we don't merge this, we should cherry pick those commits onto master.

**Comment by rachitnigam** [comment]
Pyret benchmarks should work on this now.

File: stopify-continuations/src/callcc/callcc.ts [hunk 1]
```diff
@@ -73,15 +79,21 @@ const visitor: Visitor = {
...
+    if (!(<any>opts).renames) {
+      timeSlow('delimit', () =>
+        h.transformFromAst(path, [[delimitTopLevel, opts]]));
+    }
     timeSlow('label', () =>
       h.transformFromAst(path, [label.plugin]));
     timeSlow('jumper', () =>
       h.transformFromAst(path, [[jumper.plugin, opts]]));
 
+    path.stop();
```
Comment (arjunguha) [inline stopify-continuations/src/callcc/callcc.ts]: @baxtersa I want to confirm that this is just an optimization.

Comment (baxtersa) [reply stopify-continuations/src/callcc/callcc.ts]: yes

**Comment by arjunguha** [comment]
This is now in #314, fully rebased to master.

**Pull request closed without merging**